packages/shift-next-routes/src/express/cart-handler.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
       if (response.data.id) {
         res.cookie('cart', response.data.id, {
           signed: true,
+          secure: process.env.NO_HTTPS ? false : true,
```

Comment:
This logic (or the in-browser equivalent `window.location.protocol === 'https'`) appears to be repeated all over the place... might be worthwhile extracting into one place?